example/src/main/java/com/stripe/example/activity/UpiPaymentActivity.kt

```diff
@@ -43,16 +43,18 @@ class UpiPaymentActivity : StripeIntentActivity() {
         }
     }
 
-    override fun onConfirmSuccess(result: PaymentIntentResult) {
-        val paymentIntent = result.intent
+    override fun onConfirmSuccess() {
+        //TODO(ccen) get the client secret
+        val clientSecret = "12345"
```

Comment:
@mshafrir-stripe I think it makes sense to [piggyback StripeIntent](https://github.com/stripe/stripe-android/blob/7654eaa520477022864651b3acdea289b13d9e04/payments-core/src/main/java/com/stripe/android/payments/paymentlauncher/PaymentLauncherViewModel.kt#L37) with `PaymentResult.Completed` here to avoid another round trip, wdyt?

Reply:
on a 2nd through, it's up to the merchant to save the clientSecret, as they will need to get it from their backend anyway. Refactored a bit to capture it through `StripeIntentActivity#createAndConfirmPaymentIntent`